website/versioned_docs/version-1.0.0/guides/getting_started.md

```diff
@@ -0,0 +1,1741 @@
+---
+id: version-1.0.0-getting-started
+title: Getting Started
+original_id: getting-started
+---
```

Comment:
Just to confirm. You manually created this for version 1.0.0 so that it's already available there and not only in `next` or is there something wrong with the build?

Reply:
Exactly, so the changes are in the 1.0.0 docs and later, but not in the older ones.